apiserver/handler.go

```diff
@@ -0,0 +1,230 @@
+package apiserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"istio.io/manager/model"
+
+	restful "github.com/emicklei/go-restful"
+	"github.com/golang/glog"
+)
+
+var schema model.ProtoSchema
```

Comment:
Can this be moved to line 32? It would be best to avoid use of global variables as someone will inevitably use `schema = ` instead of `schema :=` somewhere and the compiler won't complain.